maskrcnn_benchmark/config/paths_catalog.py

```diff
@@ -6,28 +6,80 @@
 
 class DatasetCatalog(object):
     DATA_DIR = "datasets"
-
```

Comment:
nit: `config/defaults.py` still have the change of permissions, can you change that?